source/administration/changelog.md

```diff
@@ -4,6 +4,171 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.6
+
+Release date: 2018-12-16
+
+### Highlights
```

Comment:
Thoughts about matching the higlights with the higlights in the blog post, where appropriate (e.g. mobile doesn't belong here, but CLI commands could be added here)?